src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/subscriptionBox.jsx

```diff
@@ -7,13 +7,16 @@ import styled from 'react-emotion';
 import Checkbox from 'app/components/checkbox';
 import Tooltip from 'app/components/tooltip';
 import {Flex} from 'grid-emotion';
+import withOrganization from 'app/utils/withOrganization';
+import SentryTypes from 'app/sentryTypes';
 
-export default class SubscriptionBox extends React.Component {
+class SubscriptionBox extends React.Component {
```

Comment:
@MeredithAnya You can always do `export class SubscriptionBox ` to avoid the extra export you have on line 74.